cli/tests/pcluster/validators/test_cluster_validators.py

```diff
@@ -2075,6 +2076,41 @@ def test_mixed_security_group_overwrite_validator(head_node_security_groups, que
     assert_failure_messages(actual_failures, expected_message)
 
 
+@pytest.mark.parametrize(
+    "shared_storage_type, shared_storage_efs_settings, expected_message",
+    [
+        (
+                "Efs",
+                {
+                    "encrypted": True
```

Comment:
Can you add the use-case of `encrypted` is False? 
so that we cover https://github.com/aws/aws-parallelcluster/pull/6914/files#r2205669568